lifecycle-application/src/test/java/io/quarkus/ts/openshift/lifecycle/LifecycleApplicationOpenShiftIT.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.ts.openshift.lifecycle;
+
+import io.fabric8.kubernetes.api.model.PodList;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.quarkus.ts.openshift.common.OpenShiftTest;
+import io.quarkus.ts.openshift.common.injection.TestResource;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static io.restassured.RestAssured.when;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@OpenShiftTest
+public class LifecycleApplicationOpenShiftIT {
+
+    private static final String LOGGING_PROPERTY = "-Djava.util.logging.manager=org.jboss.logmanager.LogManager";
+    private static final String RUNTIME_LABEL = "app.openshift.io/runtime";
+    private static final String RUNTIME_QUARKUS = "quarkus";
+
+    @TestResource
+    private OpenShiftClient openshiftClient;
+
+    @Test
+    public void shouldArgumentsNotContainLoggingProperty() {
+        when().get("/args")
+        .then().statusCode(200)
+                .body(not(containsString(LOGGING_PROPERTY)));
+    }
+
```

Comment:
Should we have tests that ensures certain format of the executed command?
That test would be dependent on the use mode - JVM or Native.
We would probably check bind address definition and logging property.

Well, how does this app behave in native mode? How are args differentiated from quarkus `-D` settings?

Reply:
So I checked and `target/lifecycle-application-1.0.0-SNAPSHOT-runner -Daaa=bbb foo bar` translates to `Received arguments: foo,bar`.

So anything in `-Dfoo=bar` format get transformed as property and is not passed further as argument for the application.

Reply:
ATM there is no direct demand to ensure certain format of the executed command. I think we can leave it as possible enhancement for the future.

Reply:
This makes sense but it's now a limitation when deploying into OpenShift as we can't provide arguments now :/